mailscanner/lib/lineConfig.js

```diff
@@ -94,6 +94,7 @@ function printLineGraph(chartId, settings) {
               showLine: (typeof settings.types === 'undefined' || settings.fillBelowLine[i] ? true :
                           (typeof settings.drawLines === 'undefined' ? false : settings.drawLines)),
               pointRadius: 1,
+              borderWidth: 0.1,
```

Comment:
But it actually shows pretty good that you only have some minutes were you are receiving mails so you can say it is at is expected 😁